cloud/docker/docker_image.py

```diff
@@ -371,12 +371,6 @@ def push_image(self, name, tag=None):
         registry, repo_name = auth.resolve_repository_name(repository)
 
         if re.search('/', repository):
-            if registry:
```

Comment:
i would not eliminate this block, but just the error on load, keep a flag and then return it if/when it fails to connect.